sass/_icon-font.scss

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: 'appbox-icons';
+  src:url('../fonts/icomoon.eot?-k1gnej');
+  src:url('../fonts/icomoon.eot?#iefix-k1gnej') format('embedded-opentype'),
+  url('../fonts/icomoon.ttf?-k1gnej') format('truetype'),
+  url('../fonts/icomoon.woff?-k1gnej') format('woff'),
+  url('../fonts/icomoon.svg?-k1gnej#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'appbox-icons';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  margin: auto;
+  width: 100%;
+  text-align: center;
+}
+
+$icons:
+  (star '\e601')
+  (star-outline '\e614') 
+  (seek '\e603')
+  (play '\f04b')
+  (Googleplay '\f04b')
```

Comment:
Because you're using these in classes, I'd recommend sticking with all lower-case naming.